gulpfile.js

```diff
@@ -366,7 +376,7 @@ function createTasksForBuildJs ({ rootDir, taskPrefix, bundleTaskOpts, destinati
       label: jsFile,
       filename: `${jsFile}.js`,
       filepath: `${rootDir}/${jsFile}.js`,
-      externalDependencies: jsFile === 'ui' && !bundleTaskOpts.devMode && uiDependenciesToBundle,
+      externalDependencies: bundleTaskOpts.devMode ? undefined : externalDependenciesMap[jsFile],
```

Comment:
We will need to apply this in dev mode too - otherwise we can't load the extension in dev mode (the file referenced in the manifest doesn't exist).

We could certainly remove it from the manifest in development as well, but... it seems like it'd be easier to just bundle things the same in development instead.